site/src/pages/components/radio.research.mdx

```diff
@@ -38,35 +39,35 @@ After the research there are still [discussions to be made](#discussion), where
 
 ## Anatomy
 
-<Anatomy component="Radio Button" />
+<Anatomy component="Radio" />
 
 ## Concepts
 
-<Concepts client:load component="Radio Button" />
+<Concepts client:load component="Radio" />
 
 ## Discussion
 
-At a point during the research, the scope of this study os shifted from the “radio button” to the “radio button group”, because there is no particular strong use case for a standalone “radio button”, and also because this use case is against the W3 Specification, the actual component to be considered is the “radio button group”, being composed of two or more radio buttons.
+At a point during the research, the scope of this study os shifted from the “radio” to the “radio group”, because there is no particular strong use case for a standalone “radio”, and also because this use case is against the W3 Specification, the actual component to be considered is the “radio group”, being composed of two or more radios.
 
 Also based on this fact and on the current state of the design systems, a more final anatomy was proposed.
 
 ### Proposed Anatomy - by Tag Hierarchy
```

Comment:
Let's remove proposed anatomy and just make it "Anatomy" as every spec in Open UI is effectively a proposal and as such makes this title redundant